intel-e1000e-alt-8.2.cfg

```
@@ -3,4 +3,4 @@ PACK_UUID="383ac20f-9c83-41be-85b0-9acf2aa3ab53"
 PACK_BUILD="pack.0.1"
 RPM_FILE="intel-e1000e-alt-3.8.7-1.xcpng8.2.x86_64.rpm"
 RPM_SHA256="4f3716c4bd5765b97f538b4f021d51853939df6505fc0a62aa718d532c64c5da"
-GPG_PUBKEY_FILE="RPM-GPG-KEY-XS-DDK-Test"
+GPG_PUBKEY_FILE="RPM-GPG-KEY-XS-DDK-SingleUse"
```

We probably don't need XS DDK in the key name.

```suggestion
GPG_PUBKEY_FILE="RPM-GPG-KEY-XCP-SingleUse"
```